studio/schemas/types/articlespreviewlist.ts

```diff
@@ -0,0 +1,25 @@
+import { Grid } from "react-feather";
+
+export default {
+  name: "articlespreviewlist",
+  type: "object",
+  title: "Articles preview list",
+  icon: Grid,
+  fields: [
+    // hidden field
+    {
+      name: "empty",
+      type: "boolean",
+      title: "Empty",
+      hidden: true,
+      initialValue: false,
+    },
```

Comment:
Might need some suggestion here.. Sanity object fields require at least one field. This workaround seems to work well enough, but it isn't very pretty. Is there any other type that could be more suitable??